vm/templates/script/rust-toolchain

```diff
@@ -0,0 +1,3 @@
+[toolchain]
+channel = "nightly-2024-04-17"
+components = ["llvm-tools", "rustc-dev"]
```

Comment:
NIT: it should build on stable AFAIR (this file and the other one can be deleted IMO)